packages/two_dimensional_scrollables/lib/src/tree_view/tree.dart

```diff
@@ -886,6 +886,16 @@ class _TreeViewState<T> extends State<TreeView<T>>
       if (widget.onNodeToggle != null) {
         widget.onNodeToggle!(node);
       }
+
+      // If animation is disabled or duration is zero, skip the animation
+      // and update the active nodes immediately. This ensures the tree
+      // is updated correctly when the node's children are no longer active.
+      if (widget.toggleAnimationStyle == AnimationStyle.noAnimation ||
+          widget.toggleAnimationStyle?.duration == Duration.zero) {
```

Comment:
When `widget.toggleAnimationStyle == AnimationStyle.noAnimation` is true, `widget.toggleAnimationStyle?.duration == Duration.zero` must be true right? Should we just keep the second one?

Reply:
The second is because we can do something like:
```
toggleAnimationStyle: AnimationStyle(
   curve: Curves.easeInOut,
   duration: Duration.zero,
),
```

Reply:
Right, I mean to keep the second only and remove the first check:)